ibis/sql/postgres/client.py

```diff
@@ -103,6 +113,21 @@ def database(self, name=None):
             )
             return self.database_class(name, new_client)
 
+    def schema(self, name):
+        """Create a database schema reflection object for the schema named `name`.
```

Comment:
This docstring is a tiny bit confusing. What about "Get a schema object from the current database" or something like that?